sample-lnd.conf

```diff
@@ -1415,3 +1415,17 @@ litecoin.node=ltcd
 ; for neutrino nodes as it means they'll only maintain edges where both nodes are
 ; seen as being live from it's PoV.
 ; routing.strictgraphpruning=true
+
+[sweeper]
+; Duration of the sweep batch window. The sweep is held back during the batch
+; window to allow more inputs to be added and thereby lower the fee per input.
+; A non-positive value will disable the batching(not recommended).
+; sweeper.batchwindowduration=30s
```

Comment:
Nice to allow configuring these values. Though they could also lead to configurations that are dangerous. For example with issues like https://github.com/lightningnetwork/lnd/issues/6855 people might try to turn off the sweeper to save on fees and then not be able to recover their funds when restoring from seed.

So I'm not sure if we should add either some warnings or a bit more validation around minimum safe values?

Reply:
The original doc was not clear😅. I think when the batch window duration is zero or negative, it will always fire the ticker immediately, without waiting for other inputs to be aggregated. I'm not sure if it's good behavior tho, so I add a check to ensure the duration is always positive.